### ISSUE TITLE
More Random Data Types

### ISSUE THREAD

**Issue by ajchili** [ZieIony/RandomData#6]
Hey, I have thought and looked up some other data types that could be useful. I wouldn't mind implementing them with you either 😃 

* App Versions
* Car
  * Make/Model
  * Year
  * Color
* Color
* Company
* Country Code
* File Name
* IP Address

* Support for exporting to JSON

**Comment by ZieIony** [comment]
Could you explain App Version?

There is a color generator and a database of countries. It should be easy to make a country code generator. IP addresses shouldn't be difficult as well. A more complex address generator with country context would be really nice, but you'd need a geocoder for that. Or a rather large database.

File names are interesting but you need a database for that. For cars and companies as well. Are you thinking about real companies? Or more like fake names?

JSON export isn't actually needed as you can use pretty much any JSON serializer to save generated objects to a JSON.

**Comment by ajchili** [comment]
A set of data for App Version could include [ 1.0, 2.4.1, 2.1.12 ] and so on.

Either real companies or fake ones, I don't think it really matters.

But for the other data types, can I pr into your branch after implementing them?

**Comment by ZieIony** [comment]
sure

**Comment by ajchili** [comment]
Thanks for doing the small changes after #7. I plan on making another PR later with a more abstract version of the App Version generator.
